### DISCUSSION TITLE
error? plz help

### DISCUSSION THREAD

**Discussion by anyangpiggy** [Pryaxis/TShock#2483] [Q&A] [answered]
I'm trying to make myself an admin so i can start setting up permissions and spawn and stuff.

I try to set it up and it says

"THE INITIAL SETUP SYSTEM IS DISABLED. THIS INCIDENT HAS BEEN LOGGED.
IF YOU ARE LOCKED OUT OF ALL ADMIN ACCOUNTS, ASK FOR HELP ON HTTPS://TSHOCK.CO"

**Comment by comdar16** [comment, marked as answer]
> I'm trying to make myself an admin so i can start setting up permissions and spawn and stuff.
> 
> I try to set it up and it says
> 
> "THE INITIAL SETUP SYSTEM IS DISABLED. THIS INCIDENT HAS BEEN LOGGED. IF YOU ARE LOCKED OUT OF ALL ADMIN ACCOUNTS, ASK FOR HELP ON [HTTPS://TSHOCK.CO](HTTPS://TSHOCK.CO)"

Looks like you did the /setup step already. Type in console /user group (nickname) superadmin to get your self admin perms.

**Reply by anyangpiggy** [reply]
thanks for help:)
it solved
